src/GeoTables.jl

```diff
@@ -9,9 +9,8 @@ using Tables
 using Random
 
 import Base: values
```

Comment:
Let's also remove this import here and use `Base.values` explicitly in the source.